samples/java-spring-choreography-saga-quickstart/src/it/java/user/registry/UserCreationIntegrationTest.java

```diff
@@ -108,10 +108,7 @@ public void testUserCreationFailureDueToInvalidInput() throws Exception {
 
     await()
       .ignoreExceptions()
-      // timer will fire in 3 seconds and un-reserve the email
-      // see it/resources/application.conf for the configuration
-      // we only start to polling after 3 seconds to give the timer a chance to fire
-      .between(Duration.ofSeconds(3), Duration.ofSeconds(6))
+      .timeout(Duration.ofSeconds(10)) //3 seconds for the projection lag + 3 seconds for the timer to fire
```

Comment:
@octonato for the record, using `between` has different semantics, it starts pooling immediately, but the assertion must be valid only within the boundaries.